wsi/drm/drmplane.cpp

```diff
@@ -514,10 +513,17 @@ bool DrmPlane::IsSupportedModifier(uint64_t modifier, uint32_t format) {
     const format_mods& obj = formats_modifiers_.at(i);
     if (obj.format == format) {
       std::vector<uint64_t>::const_iterator it;
+#ifdef OS_YUNOS
+      for (auto mod : obj.mods) {
```

Comment:
why is this YUNOS specific ?

Reply:
YUNOS toolchain is too old, there is compile error here.